test/functional/p2p_invalid_getdata.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Request non-existent block.
+"""
+
+from test_framework.mininode import CInv, P2PInterface, msg_getdata
+from test_framework.test_framework import BitcoinTestFramework
+
+# TestP2PConn: A peer we use to send messages to bitcoind, and store responses.
+class TestP2PConn(P2PInterface):
+    def __init__(self):
+        super().__init__()
```

Comment:
Doesn't seem like this subclass is necessary. Any reason you can't just use P2PInterface on line 23 below?

Reply:
Indeed. Please have a look at wallet_disable.py or example_test.py on how to use the test framework.

Reply:
Agree. (Sorry, my previous comment didn't apply, thus deleted)